### PR TITLE
add avtar and button

### PR DESCRIPTION
**Pull request by Divya70**
add diffrent styles of avtar and button
1. avtar with image
2.avtar with text
.  simple button
. floating button
.button with icon

avtar
button

### CODE REVIEW

File: pages/Avtar/avtar.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+.avtar-title{
+      margin: 2rem 0 0 5rem;
+      color: var(--primary-color);
+      font-size: 3rem;
+}
+.avtar-details{
+      padding: 10px;
+      color: var(--white-color);
+      font-size: 1.3rem;
+      width: 900px;
```
Comment (Ayushi20-19) [inline pages/Avtar/avtar.css:20]: maintain the consistency and use either px or rems

File: pages/Avtar/avtar.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      width: 70vw;
+      display: flex;
+    flex-wrap: wrap;
+    padding: 1rem;
+    margin: 2rem 0;
+    border: 2px solid var(--black-color);
+    align-items: baseline;
+    border-radius: 0.3rem;
+    border: 1px solid rgb(192, 192, 192);
+    margin-left: 4rem;
```
Comment (Ayushi20-19) [inline pages/Avtar/avtar.css:33]: u can shift this margin at line 28 with single line margin

File: pages/Avtar/avtar.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+}
+.text{
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background-color: var(--primary-color);
+      border-radius: 50%;
+      border: 2px solid white;
+      font-weight: 600;
+      font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
```
Comment (Ayushi20-19) [inline pages/Avtar/avtar.css:95]: so many fonts? any particular reason?

File: pages/Avtar/avtar.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    border: 2px solid var(--white-color);
+    border-radius: 0.3rem;
+}
+.text{
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      background-color: var(--primary-color);
+      border-radius: 50%;
+      border: 2px solid white;
```
Comment (Ayushi20-19) [inline pages/Avtar/avtar.css]: u have white color as variable so u can use that here also

File: pages/Button/button.css [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+        border:0;
+}
+
+/* Button with icon code */
+
+.icon-btn-title{
+      margin-left: 2rem;
+      color: var(--primary-color);
+      font-size: 3rem;
+      margin-top: -12rem;
```
Comment (Ayushi20-19) [inline pages/Button/button.css:138]: why a negative margin is required here??

File: pages/Button/button.css [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+}
+.cart-button{
+  background-color: #B6EB7A;
+  color: #17706E;
+  padding:0.7rem
+}
+.home-button{
+      padding:0.7rem
+}
+.icon-btn-iframe{
```
Comment (Ayushi20-19) [inline pages/Button/button.css:163]: avoid using other CSS than the button in the button.css file u can make a common class for this and use in doc page and import form there.

**Review by Ayushi20-19 — CHANGES_REQUESTED** [review]
Avoid using multiple units go with either rem of px, don't use other CSS other than that component CSS in the component CSS file.

**Comment by gadgetfather** [comment]
> add diffrent styles of avtar and button
> 
> 1. avtar with image
>    2.avtar with text
>    .  simple button
>    . floating button
>    .button with icon
> 
> avtar button

You can align those images in center

File: pages/Button/button.css [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+      font-size: 3rem;
+      margin-top: -12rem;
+}
+
+.icon-button-cont{
+      padding: 3.5rem;
+      margin-top: -3rem;
+}
+.icon-btn{
+      /* padding: 1rem; */
```
Comment (gadgetfather) [inline pages/Button/button.css]: Remove this if you are not using this

File: pages/Button/button.css (hunk 6 above)
Comment (Divya70) [reply pages/Button/button.css:163]: i will separate file later

File: pages/Button/button.css [hunk 8]
```diff
@@ -0,0 +1,169 @@
...
+      height: 20vh;
+      margin: 2rem;
+      border: 1px solid rgb(192, 192, 192);
+}
+.button{
+      padding: 0.6rem;
+      margin: 1rem;
+      font-size: 1rem;
+      font-weight: 600;
+      box-shadow: 3px;
```
Comment (gadgetfather) [inline pages/Button/button.css:36]: please check this box-shadow property

**Review by Ayushi20-19 — APPROVED** [review]

**Review by gadgetfather — APPROVED** [review]

**Pull request merged**